src/types/annotation.ts

```diff
@@ -1,13 +1,27 @@
-export interface PostAnnotationResponse {
-  stream: string
-  endTime: string
-  startTime: string
+export interface Annotation {
+  end: string
+  start: string
```

Comment:
giraffe uses startValue and stopValue

here we are using start and end

inconsistencies like this are not good.

Reply:
Per @hoorayimhelping this is something that will be handled in the view layer not the api layer.